doc/sample_repos.md

```diff
@@ -191,26 +210,45 @@ RStudio support with JupyterHub*
 ### Files
 ```
 Dockerfile
+How to Launch RStudio.ipynb
 index.R
 ```
+```eval_rst
+|
 
-
-## Python 2 with `runtime.txt`
+```
+---------
+## Specifying a Python 2 environment with `runtime.txt`
 
 [![Binder](http://mybinder.org/badge.svg)](http://mybinder.org/v2/gh/binder-examples/python2_runtime/master) | [repo link](https://github.com/binder-examples/python2_runtime)
 
 We can specify various runtime parameters with a `runtime.txt` file. In this
-repository, we demonstrate how to install python 2 with this environment.
+repository, we demonstrate how to install python 2 with the environment.
+
+If you specify `python-2.7` in `runtime.txt`, then:
+
+* A python3 environment is created & installed (this is what the notebook runs from)
+* A python2 environment is created and registered
+* The contents of `requirements.txt` are installed into the python2 environment
+
+Note:
+* Make sure that you save your notebooks with a python 2 kernel activated,
+  as this defines which kernel Binder will use when a notebook is opened.
+* If you *also* wish to install python 3 dependencies, you may do so by including a
```

Comment:
Let's make the first note "important" (must do to have it work) and the second note a "note" (optional)